### PR TITLE
Add zip functionality

### PR DESCRIPTION
**Pull request by sujaygarlanka**
Added the ability to create a zip file in Box and download it.

Added a unit test and did manual testing.

### CODE REVIEW

File: src/commands/files/zip.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        let fileName = args.name;
+        if (!fileName.includes('.zip')) {
+            fileName += '.zip';
+        }
+
+		let filePath = path.join(flags.destination || this.settings.boxDownloadsFolderPath, fileName);
+        
+        /* eslint-disable no-sync */
+		if (fs.existsSync(filePath)) {
+		/* eslint-enable no-sync */
```
Comment (PJSimon) [inline src/commands/files/zip.js:22]:
```suggestion
		/* eslint-disable no-sync */
		if (fs.existsSync(filePath)) {
		/* eslint-enable no-sync */
```

File: src/commands/files/zip.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        }
+ 
+        let output = fs.createWriteStream(filePath);
+        let downloadStatus = await this.client.files.downloadZip(fileName, flags.item, output);
+		await this.output(downloadStatus);
+	}
+}
+
+FilesZipCommand.description = 'Create a zip of multiple files and folders and download it';
+FilesZipCommand.examples = ['box files:zip sample_file.zip --item=file:12421 --item=folder:48291'];
```
Comment (PJSimon) [inline src/commands/files/zip.js:38]: Can `--item=` take a comma-delimited string of item references instead of/in addition to this?

Comment (sujaygarlanka) [reply src/commands/files/zip.js:38]: Ya, can change it to a comma if you think it makes more sense.

Comment (PJSimon) [reply src/commands/files/zip.js]: Sorry @sujaygarlanka, I meant to change it to `--items=file:12421,folder:48291` which seems more concise.
Do you think that would make more sense?  I feel like I've seen args done this way more often (in general).  But preference would be to do what is consistent in this app.

**Pull request merged**